src/photos.cs

```diff
@@ -216,11 +225,11 @@ interface PHAssetCreationRequest
 
 	[iOS (9,0)]
 	[TV (10,0)]
-	[NoMac]
-	[BaseType (typeof(NSObject))]
+	[Mac (10,15, onlyOn64: true)]
+	[BaseType (typeof (NSObject))]
```

Comment:
Unnecessary whitespace change:

```suggestion
	[BaseType (typeof(NSObject))]
```